runtime/onert/core/include/ir/operation/StatelessRandomUniform.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_IR_OPERATION_STATELESS_RANDOM_UNIFORM_H__
+#define __ONERT_IR_OPERATION_STATELESS_RANDOM_UNIFORM_H__
+
+#include <memory>
+
+#include "ir/Operation.h"
+
+namespace onert
+{
+namespace ir
+{
+namespace operation
+{
+
+class StatelessRandomUniform : public Operation
+{
+public:
+  enum Input
+  {
+    INPUT = 0,
+    SEED = 1
```

Comment:
@YiHyunJin Please check what would be tflite's input and property of `stateless_uniform` after reading [tf.random.stateless_uniform](https://www.tensorflow.org/api_docs/python/tf/random/stateless_uniform). You seem to read the input from `tf`.

- Our input could be `seed` and `min`, `max`:
- `seed` could be as input tensor as you wrote.
- I cannot find `min` and `max`, which are passed by tf attributes.
  - Note the data type of `min` and `max` could be `float32` or other types.
- Current `INPUT` is the shape of `input` in `tf` world. We may use this information to determine output tensor's shape.

Reply:
okay i will check.

Reply:
@YiHyunJin As you informed me this morning, the default value for `min` and `max` is `0` and `1`. We may provide the simplest implementation without `min` and `max` since our current model requires (min, max) = (very small positive number, 1).